telco-examples/edge-clusters/README.md

```diff
@@ -63,16 +49,25 @@ There are two different examples:
 - [Example 1 - Deploy a single-node Edge Cluster with the image generated and Telco profiles](#example-1---deploy-a-single-node-edge-cluster-with-the-image-generated-and-telco-profiles): In this example we will demostrate how to deploy a single-node edge cluster using the image generated in the previous step and the telco profiles configured in order to use telco capabilities like SRIOV, DPDK, CPU pinning and so on.
 - [Example 2 - Deploy a multi-node HA cluster using metal3, metallb and the image generated](#example-2---deploy-a-multi-node-ha-cluster-using-metal3-metallb-and-the-image-generated): In this example we will demostrate how to deploy a multi-node edge cluster using metal3, metallb and the image generated in the previous step.
 
+For both examples, there are 2 options available to be used depending on your environment:
+
+- DHCP Environment: In case you are using a DHCP environment, you need to use the `telco-examples/edge-clusters/dhcp` folder.
+- DHCP-less Environment: In case you are using a DHCP-less environment, you need to use the `telco-examples/edge-clusters/dhcp-less` folder.
```

Comment:
As we discussed yesterday - I think we can actually use the same manifests and remove the Metal3DataTemplate networkData in both cases, and we could re-word this to describe "additional network config" instead of DHCP-less

Should we do that now or as a follow-up?